plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestJdbcClient.java

```diff
@@ -116,7 +116,7 @@ public void testMetadataWithFloatAndDoubleCol()
     @Test
     public void testCreateSchema()
     {
-        String schemaName = "mixed case schema";
+        String schemaName = "test schema";
```

Comment:
wrong commit? (part of `Table rename connector (smoke) test`)

Reply:
Sorry, fixed.